pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -605,12 +610,18 @@ class Build {
             // Primary platforms run extended.jck in Parallel
             targets['serial']   = 'sanity.jck,special.jck'
             targets['parallel'] = 'extended.jck'
+            if ( !Boolean.valueOf(buildConfig.WEEKLY) ) {
+                targets['serial_dev']   = 'dev.jck'
+            }
```

Comment:
This needs moving outside of this if, otherwise it is only running for x64 linux/win/mac